wagtail/wagtailcore/models.py

```diff
@@ -257,7 +257,7 @@ class PageBase(models.base.ModelBase):
     def __init__(cls, name, bases, dct):
         super(PageBase, cls).__init__(name, bases, dct)
 
-        if cls._deferred:
+        if hasattr(cls, '_deferred'):
```

Comment:
I think that it will not work. I've found some useful docs:
* https://docs.djangoproject.com/en/1.10/ref/models/instances/#customizing-model-loading
* https://docs.djangoproject.com/en/1.9/ref/models/instances/#customizing-model-loading